cosmogony/Cargo.lock

```diff
@@ -0,0 +1,633 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This `cosmogony/Cargo.lock` is not supposed to be committed, right ? As `cosmgony/` is used to publish the `cosmogony` crate.

Reply:
My question still stands.

Reply:
yep, sorry, completly forgot this :/